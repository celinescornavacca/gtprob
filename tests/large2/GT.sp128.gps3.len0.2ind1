((((EA0,(EA1,EA2)),(((VC1,VC2),VC0),(WC0,(WC1,WC2)))),((((MC1,MC2),MC0),((QE0,(QE1,QE2)),(RE0,(RE1,RE2)))),(MA0,(MA1,MA2)))),((((((((UD1,UD2),UD0),((VD1,VD2),VD0)),(YC0,(YC1,YC2))),(((NB0,(NB1,NB2)),((((ED1,ED2),ED0),(((OD1,OD2),OD0),(PD0,(PD1,PD2)))),((XC0,(XC1,XC2)),((FD0,(FD1,FD2)),((GD1,GD2),GD0))))),((AA1,AA2),AA0))),((((((QD1,QD2),QD0),(RD0,(RD1,RD2))),(DD0,(DD1,DD2))),(((ID0,(ID1,ID2)),((JD1,JD2),JD0)),(((WD1,WD2),WD0),((XD1,XD2),XD0)))),(((((ZB1,ZB2),ZB0),((AC1,AC2),AC0)),((AB1,AB2),AB0)),((FA1,FA2),FA0)))),(((((KC0,(KC1,KC2)),(LC0,(LC1,LC2))),(QB0,(QB1,QB2))),(((CC0,(CC1,CC2)),((MD0,(MD1,MD2)),((ME0,(ME1,ME2)),((NE1,NE2),NE0)))),(((SB1,SB2),SB0),((TB1,TB2),TB0)))),((((((UB1,UB2),UB0),(((KD1,KD2),KD0),(LD0,(LD1,LD2)))),((((SE1,SE2),SE0),(TE0,(TE1,TE2))),((DC1,DC2),DC0))),((JB0,(JB1,JB2)),((KB1,KB2),KB0))),(CA0,(CA1,CA2))))),((((GC0,(GC1,GC2)),((HC1,HC2),HC0)),((((CB0,(CB1,CB2)),((DB1,DB2),DB0)),(ZA0,(ZA1,ZA2))),((OA1,OA2),OA0))),((((((TC1,TC2),TC0),((UC1,UC2),UC0)),((((PC0,(PC1,PC2)),((QC1,QC2),QC0)),((((VA1,VA2),VA0),(WA0,(WA1,WA2))),((((OB1,OB2),OB0),(PB0,(PB1,PB2))),((HB1,HB2),HB0)))),(((((NA1,NA2),NA0),((UA0,(UA1,UA2)),(((EB1,EB2),EB0),((FB1,FB2),FB0)))),((((RA0,(RA1,RA2)),(SA0,(SA1,SA2))),((BC0,(BC1,BC2)),(((EC1,EC2),EC0),((FC1,FC2),FC0)))),(((LB0,(LB1,LB2)),((MB1,MB2),MB0)),(BB0,(BB1,BB2))))),((((IC0,(IC1,IC2)),(JC0,(JC1,JC2))),(((GB1,GB2),GB0),((IB0,(IB1,IB2)),((CE0,(CE1,CE2)),(DE0,(DE1,DE2)))))),((XB0,(XB1,XB2)),(YB0,(YB1,YB2))))))),((((((BA1,BA2),BA0),((((IE1,IE2),IE0),((JE1,JE2),JE0)),(((ZC1,ZC2),ZC0),(AD0,(AD1,AD2))))),((DA0,(DA1,DA2)),(((GA1,GA2),GA0),(HA0,(HA1,HA2))))),(((WE0,(WE1,WE2)),(XE0,(XE1,XE2))),((SD0,(SD1,SD2)),((TD1,TD2),TD0)))),(((((((GE1,GE2),GE0),((HE1,HE2),HE0)),(VB0,(VB1,VB2))),((TA1,TA2),TA0)),(((KE0,(KE1,KE2)),(LE0,(LE1,LE2))),(((((YD1,YD2),YD0),((OE0,(OE1,OE2)),((PE1,PE2),PE0))),(((ZD1,ZD2),ZD0),((UE0,(UE1,UE2)),(VE0,(VE1,VE2))))),((WB1,WB2),WB0)))),((((XA1,XA2),XA0),(YA0,(YA1,YA2))),((JA0,(JA1,JA2)),((KA1,KA2),KA0)))))),(((((RB1,RB2),RB0),((((ND1,ND2),ND0),(((EE1,EE2),EE0),((FE1,FE2),FE0))),((HD1,HD2),HD0))),(((((RC1,RC2),RC0),((SC1,SC2),SC0)),((NC0,(NC1,NC2)),(OC0,(OC1,OC2)))),(((BD0,(BD1,BD2)),(CD0,(CD1,CD2))),(((AE0,(AE1,AE2)),((BE1,BE2),BE0)),(LA0,(LA1,LA2)))))),((((PA1,PA2),PA0),((QA1,QA2),QA0)),(IA0,(IA1,IA2))))))));
