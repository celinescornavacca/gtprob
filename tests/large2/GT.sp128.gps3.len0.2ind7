(((((BA1,BA2),BA0),(((ZC0,(ZC1,ZC2)),(AD0,(AD1,AD2))),((MA0,(MA1,MA2)),((AB0,(AB1,AB2)),(BB0,(BB1,BB2)))))),(((OC0,(OC1,OC2)),(((BD1,BD2),BD0),(((ED1,ED2),ED0),(FD0,(FD1,FD2))))),(DA0,(DA1,DA2)))),((((((JB0,(JB1,JB2)),(((QD1,QD2),QD0),((RD1,RD2),RD0))),((((CE0,(CE1,CE2)),((DE1,DE2),DE0)),(((SD0,(SD1,SD2)),(TD0,(TD1,TD2))),(GD0,(GD1,GD2)))),((((KC0,(KC1,KC2)),(LC0,(LC1,LC2))),(((OD1,OD2),OD0),((PD1,PD2),PD0))),((SA1,SA2),SA0)))),(((((OE0,(OE1,OE2)),((PE1,PE2),PE0)),((WB1,WB2),WB0)),(((HA1,HA2),HA0),((IA1,IA2),IA0))),(((HD1,HD2),HD0),((UD0,(UD1,UD2)),((VD1,VD2),VD0))))),((((((WE1,WE2),WE0),((XE1,XE2),XE0)),((QC1,QC2),QC0)),(((CB1,CB2),CB0),(((ID0,(ID1,ID2)),((JD1,JD2),JD0)),(((EC0,(EC1,EC2)),((FC1,FC2),FC0)),(((SE0,(SE1,SE2)),((TE1,TE2),TE0)),((PC1,PC2),PC0)))))),((((HB0,(HB1,HB2)),((((YD0,(YD1,YD2)),((ZD1,ZD2),ZD0)),((PB1,PB2),PB0)),(IB0,(IB1,IB2)))),((JA1,JA2),JA0)),((KE0,(KE1,KE2)),(LE0,(LE1,LE2)))))),(((((((WA0,(WA1,WA2)),(((IE1,IE2),IE0),((JE1,JE2),JE0))),((EA1,EA2),EA0)),((((((((GC1,GC2),GC0),((MC0,(MC1,MC2)),(NC0,(NC1,NC2)))),(XA0,(XA1,XA2))),(UA0,(UA1,UA2))),((((HC0,(HC1,HC2)),((IC1,IC2),IC0)),((TB0,(TB1,TB2)),((UB1,UB2),UB0))),(((TC1,TC2),TC0),(UC0,(UC1,UC2))))),((GB0,(GB1,GB2)),(((YB1,YB2),YB0),((ZB1,ZB2),ZB0)))),(CA0,(CA1,CA2)))),((((((((KB1,KB2),KB0),((LB1,LB2),LB0)),((EB1,EB2),EB0)),((YA1,YA2),YA0)),(TA0,(TA1,TA2))),(((VA1,VA2),VA0),(((VC1,VC2),VC0),((WC1,WC2),WC0)))),((NA0,(NA1,NA2)),((OA1,OA2),OA0)))),((((((((((WD1,WD2),WD0),(XD0,(XD1,XD2))),((KD0,(KD1,KD2)),((LD1,LD2),LD0))),(VB0,(VB1,VB2))),(((MD1,MD2),MD0),(ND0,(ND1,ND2)))),(SB0,(SB1,SB2))),((CD0,(CD1,CD2)),((DD1,DD2),DD0))),((KA0,(KA1,KA2)),(((QA1,QA2),QA0),((RA1,RA2),RA0)))),(((((MB1,MB2),MB0),((AC0,(AC1,AC2)),((FE0,(FE1,FE2)),((GE0,(GE1,GE2)),(HE0,(HE1,HE2)))))),(((NB1,NB2),NB0),(OB0,(OB1,OB2)))),((((XB0,(XB1,XB2)),((RC0,(RC1,RC2)),((SC1,SC2),SC0))),((GA1,GA2),GA0)),(((FB1,FB2),FB0),((CC0,(CC1,CC2)),((DC1,DC2),DC0))))))),(((AA1,AA2),AA0),(((QB0,(QB1,QB2)),((RB1,RB2),RB0)),(((((DB0,(DB1,DB2)),((((ME1,ME2),ME0),((NE1,NE2),NE0)),((EE1,EE2),EE0))),((ZA1,ZA2),ZA0)),(LA0,(LA1,LA2))),((((((((AE1,AE2),AE0),(BE0,(BE1,BE2))),(((QE1,QE2),QE0),(RE0,(RE1,RE2)))),(JC0,(JC1,JC2))),(((XC0,(XC1,XC2)),((YC1,YC2),YC0)),((BC1,BC2),BC0))),((FA1,FA2),FA0)),(((PA1,PA2),PA0),((UE0,(UE1,UE2)),(VE0,(VE1,VE2)))))))))));
