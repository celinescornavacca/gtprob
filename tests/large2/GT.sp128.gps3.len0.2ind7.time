62.01user 0.82system 1:03.07elapsed 99%CPU (0avgtext+0avgdata 119140maxresident)k
150inputs+0outputs (0major+395827minor)pagefaults 0swaps
