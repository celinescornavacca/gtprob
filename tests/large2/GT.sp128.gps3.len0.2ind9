(((((((QB1,QB2),QB0),(RB0,(RB1,RB2))),(((((YB1,YB2),YB0),((ZB1,ZB2),ZB0)),(((WE0,(WE1,WE2)),((XE1,XE2),XE0)),(HE0,(HE1,HE2)))),(((MB1,MB2),MB0),((NB1,NB2),NB0)))),(((((((WC0,(WC1,WC2)),((XC1,XC2),XC0)),(((IE1,IE2),IE0),(JE0,(JE1,JE2)))),(EC0,(EC1,EC2))),((AC0,(AC1,AC2)),(BC0,(BC1,BC2)))),(((FB1,FB2),FB0),(GB0,(GB1,GB2)))),(((PB1,PB2),PB0),((((YD1,YD2),YD0),(ZD0,(ZD1,ZD2))),(LC0,(LC1,LC2)))))),(((((MC1,MC2),MC0),(NC0,(NC1,NC2))),(((UC0,(UC1,UC2)),((VC1,VC2),VC0)),(((FE0,(FE1,FE2)),(GE0,(GE1,GE2))),(((QE1,QE2),QE0),((RE1,RE2),RE0))))),((((((TC1,TC2),TC0),((TD0,(TD1,TD2)),((UD1,UD2),UD0))),((AB0,(AB1,AB2)),((BB1,BB2),BB0))),(AA0,(AA1,AA2))),(((PA0,(PA1,PA2)),(((HD1,HD2),HD0),((ID1,ID2),ID0))),((((((FC1,FC2),FC0),(GC0,(GC1,GC2))),(KA0,(KA1,KA2))),(IA0,(IA1,IA2))),((((HB0,(HB1,HB2)),((((HC1,HC2),HC0),((IC1,IC2),IC0)),((UB1,UB2),UB0))),((UA1,UA2),UA0)),((QA0,(QA1,QA2)),(((SB1,SB2),SB0),(TB0,(TB1,TB2)))))))))),((((((FA1,FA2),FA0),((((WD1,WD2),WD0),((XD1,XD2),XD0)),(((CD1,CD2),CD0),(DD0,(DD1,DD2))))),(((NA1,NA2),NA0),(((((UE1,UE2),UE0),(VE0,(VE1,VE2))),((CE1,CE2),CE0)),(VA0,(VA1,VA2))))),((((((EA0,(EA1,EA2)),(((JB1,JB2),JB0),(KB0,(KB1,KB2)))),(DA0,(DA1,DA2))),((RD0,(RD1,RD2)),(SD0,(SD1,SD2)))),(((QC1,QC2),QC0),(((SE1,SE2),SE0),((TE1,TE2),TE0)))),((CB0,(CB1,CB2)),(DB0,(DB1,DB2))))),((((CA1,CA2),CA0),(((OC0,(OC1,OC2)),(((MD1,MD2),MD0),(ND0,(ND1,ND2)))),((GA1,GA2),GA0))),(((((((RC0,(RC1,RC2)),(SC0,(SC1,SC2))),((IB1,IB2),IB0)),((((PD1,PD2),PD0),((QD1,QD2),QD0)),((OE0,(OE1,OE2)),((PE1,PE2),PE0)))),((((((((KE0,(KE1,KE2)),((LE1,LE2),LE0)),((VD1,VD2),VD0)),((VB1,VB2),VB0)),(((YC1,YC2),YC0),(ZC0,(ZC1,ZC2)))),(JA0,(JA1,JA2))),(((((YA1,YA2),YA0),((ZA1,ZA2),ZA0)),((((AD1,AD2),AD0),((BD1,BD2),BD0)),((WA1,WA2),WA0))),(OA0,(OA1,OA2)))),((((PC1,PC2),PC0),((JD0,(JD1,JD2)),(((ME1,ME2),ME0),((NE1,NE2),NE0)))),(((JC1,JC2),JC0),(KC0,(KC1,KC2)))))),((((LA1,LA2),LA0),(MA0,(MA1,MA2))),((((((FD0,(FD1,FD2)),(GD0,(GD1,GD2))),((((KD1,KD2),KD0),((LD1,LD2),LD0)),(ED0,(ED1,ED2)))),((TA1,TA2),TA0)),((BA1,BA2),BA0)),((((OB1,OB2),OB0),((OD0,(OD1,OD2)),(((DE1,DE2),DE0),((EE1,EE2),EE0)))),((((RA1,RA2),RA0),(SA0,(SA1,SA2))),((((WB0,(WB1,WB2)),(XB0,(XB1,XB2))),(LB0,(LB1,LB2))),((EB1,EB2),EB0))))))),((HA0,(HA1,HA2)),((((CC1,CC2),CC0),(((AE0,(AE1,AE2)),(BE0,(BE1,BE2))),((DC1,DC2),DC0))),(XA0,(XA1,XA2))))))));
