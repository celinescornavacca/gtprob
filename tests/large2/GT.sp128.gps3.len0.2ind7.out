-607.010322334
